project/Settings.scala

```diff
@@ -42,7 +42,8 @@ object Settings {
       organization  := "org.broadinstitute.dsde.workbench",
       scalaVersion  := "2.12.6",
       resolvers ++= commonResolvers,
-      scalacOptions ++= commonCompilerSettings
+      scalacOptions ++= commonCompilerSettings,
+      addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full)
```

Comment:
I don't know if this is the right place to put this. Any sbt experts out there?

Reply:
I think this is the correct place, in project settings. Dare I ask why you need macro paradise? :) Personally I try and steer clear of using Scala macros, but I'm not a macro expert.